stories/message-box/message-box.stories.js

```diff
@@ -51,12 +52,13 @@ Note: Include two action buttons in the message box when the user's decision is
 
 const messageBoxHeight = 200;
 
-export const structure = () => `<div class="fd-message-box-docs-static fd-message-box fd-message-box--active">
```

Comment:
Don't remove `fd-message-box-docs-static`, It's needed there. I would add `z-index: 1` to this class, not to the lib